src/transform/transform.ts

```diff
@@ -573,6 +569,9 @@ function createResponseType(codeModel: CodeModel, group: OperationGroup, op: Ope
       const pagers = <Array<PagerInfo>>codeModel.language.go!.pageableTypes;
       for (const pager of values(pagers)) {
         if (pager.name === name) {
+          if (isLROOperation(op)) {
+            pager.respField = true;
```

Comment:
Is this really necessary?  Seems like we should have created the `PagerInfo` with this properly set.

Reply:
Ah, this is the case where the pager is shared between sync/async APIs.  Probably worth adding a comment to explain this.

Reply:
Done